src/pages/CreateToken.vue

```diff
@@ -34,6 +36,13 @@ export default {
 
   methods: {
     async onSubmit(e) {
+      // Check network
+      let chainID = await web3.eth.getChainId();
+      if (chainID !== 22 && chainID !== 2828) {
```

Comment:
Don't Repeat Yourself

The moment you use the same variable at a 2nd place in your code -> use a common variable.

You don't want to update these values 10 times if they change.